### PR TITLE
Script doc fixes

### PR DESCRIPTION
**Pull request by zas**
# Summary

* This is a…
    * [ ] Bug fix
    * [ ] Feature addition
    * [ ] Refactoring
    * [x] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem

* JIRA ticket (_optional_): PICARD-XXX

# Solution

# Action

### CODE REVIEW

File: picard/script.py [hunk 1]
```diff
@@ -1165,7 +1182,8 @@ def func_is_complete(parser):
 @script_function(documentation=N_(
     """`$firstalphachar(text,nonalpha="#")`
 
-Returns the first character of `text`. If `text` is not an alphabetic character `nonalpha` is returned instead.
+Returns the first character of `text`.
+If `text` is not an alphabetic character `nonalpha` is returned instead, if not passed `#`.
```
Comment (phw) [inline picard/script.py:1186]: I find the part ", if not passed #" hard to understand. What about "..., which defaults to #" or maybe more explicit "If nonalpha is not passed, # is used."?

File: picard/script.py [hunk 2]
```diff
@@ -1375,16 +1388,14 @@ def func_ne_any(parser, x, *args):
 
 Returns `text` in title case (first character in every word capitalized).
 
+Example:
+
+    $set(album,$title(%album%))
 _Since Picard 2.1_"""
-))
+), debug_doc=True)
```
Comment (phw) [inline picard/script.py:1395]: Is debug_doc set by purpose?

Comment (zas) [reply picard/script.py:1395]: Nope, leftover from experiment, I'll clean this.

File: picard/script.py (hunk 1 above)
Comment (zas) [reply picard/script.py]: I addressed this and other issues, rebased.

**Comment by zas** [comment]
I just removed useles backquotes on an example

**Comment by rdswift** [comment]
I see that you've changed 'len' to 'length' for clarity.  Is it worthwhile to also change 'num' to 'number'?

**Comment by zas** [comment]
> I see that you've changed 'len' to 'length' for clarity. Is it worthwhile to also change 'num' to 'number'?

Yes, good suggestion, done in c7477c1

**Pull request merged**